src/System.IO.Compression.ZipFile/pkg/System.IO.Compression.ZipFile.pkgproj

```diff
@@ -1,10 +1,7 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <ProjectReference Include="..\ref\System.IO.Compression.ZipFile.csproj">
```

Comment:
Yeah, I'm equally worried given how many times this PR has failed due to that specific issue.  I think I'm going to scrap removing the csproj.  Instead I'll take the part of my tool that calculates this and put it in the package build.